New chapters (#312)
infinum/eightshift-docs: website/forms/php/shortcodes.md

---
id: shortcodes
title: Shortcodes
---

## What are shortcodes?

> Shortcodes are small pieces of code that you can use to add predefined forms to your website. They are used to simplify the process of adding forms to your website.

Reviewer: ```suggestion
Shortcodes are small pieces of code that allow adding forms to the site in places blocks can't be easily added.
```

> Shortcodes are small pieces of code that you can use to add predefined forms to your website. They are used to simplify the process of adding forms to your website.

Reviewer 2: ```suggestion
Shortcodes are snippets of code which facilitate adding forms to the site in situations blocks can't be easily added.
```

> ## esFormsLink

Reviewer: ```suggestion
## `esFormsLink`
```

> This shortcode is used to add a simple link to your form. For example you need to add a privacy policy link to your form checkbox label.

Reviewer: ```suggestion
This shortcode allows adding links within text labels, e.g. if you need to add a privacy policy link to a form checkbox label.
```

> ```php

Reviewer: switch `php` to nothing 😄 (suggest doesn't work)

[esFormsLink url="https://eightshift.com" label="Eightshift" /]

> ```

Reviewer 3: ~~~suggestion
```
[esFormsLink url="https://eightshift.com" label="Eightshift" /]
```
~~~

@goranalkovic-infinum you can use `~~~` instead of backticks when you want to wrap something that has a backtics in the suggestions.

> ## esFormsROIP

Reviewer: ```suggestion
## `esFormsROIP`
```

> This shortcode is used to toggle content in the result output block based on the API response. If you API response returns a specific key it will use that value and replace the default content.

Reviewer: ```suggestion
The `esFormsROIP` (_Result output item part_) shortcode allows showing certain content inline, based on the user input. The content is shown if the provided `name` matches the fields defined within filters.
```

> This shortcode is used to toggle content in the result output block based on the API response. If you API response returns a specific key it will use that value and replace the default content.

Reviewer 2: ```suggestion
This shortcode is used to toggle content in the result output block based on the API response. If the API response returns a specific key it will use that value and replace the default content.
```
